python/guillotina/pyproject.toml

```
@@ -7,5 +7,5 @@ requires = ["flit_core"]
 name='server'
 version = '1.0.0'
 description = "Simple benchmark implementation"
-dependencies = ["guillotina>=7,<7.1","zope.interface<6.0"]
+dependencies = ["guillotina>=7,<7.1","zope.interface<7.1"]
```

```suggestion
dependencies = ["guillotina>=7,<7.1"]
```

@masipcat @bloodbare is it ok not to specify zope interface version ?